pkg/cmd/roachtest/roachtestutil/mixedversion/mixedversion.go

```diff
@@ -750,6 +750,11 @@ func (t *Test) Run() {
 
 	t.logger.Printf("mixed-version test:\n%s", plan.PrettyPrint())
 
+	// Mark the deployment mode and versions, so they show up in the github issue. This makes
+	// it easier to group failures together without having to dig into the test logs.
+	t.rt.AddParam("MVT_DEPLOYMENT_MODE", string(plan.deploymentMode))
```

Comment:
Super Nit: These look a bit different `MVT_DEPLOYMENT_MODE` vs `mvtDeploymentMode` as the other parameters are camelCase. Although we have the `ROACHTEST_` prefix, so I'm not sure if there's a certain style to the parameters here.

Reply:
What do you think about `MVT_deploymentMode` and then we don't add the prefix? I'd argue the `ROACHTEST_` prefix is redundant if we're just going to append it to everything so at least this way we can differentiate between roachtest framework parameters and test specific parameters.

Reply:
>  I'd argue the ROACHTEST_ prefix is redundant if we're just going to append it to everything so at least this way we can differentiate between roachtest framework parameters and test specific parameters.

Yeah, I was thinking about the same thing when I wrote my other comment. I do think maybe we should drop `ROACHTEST_`; I don't think anyone really cares about it.

Reply:
Okay, no prefix + camel case sounds good to me 👍